explorer/src/components/common/Address.tsx

```diff
@@ -52,6 +54,10 @@ export function Address({
     addressLabel = addressLabel.slice(0, truncateChars) + "…";
   }
 
+  if (overrideText) {
+    addressLabel = overrideText;
+  }
+
```

Comment:
Let's make this API more React-y! Accept the override label as `children`.

```
<Address pubkey={ix.programId}>
  Override text!
</Address>
```

Then…

```
function Address({children, ...props}: {children?: string, ...}) {
  const addressLabel = children ?? /* default label */;
  /* ... */
}
```

That way, the `Address` component is just a wrapper that either adds functionality around a user-supplied display string or prepares a default display string.

Reply:
Sounds good to me!